src/main/views/task-list.njk

```diff
@@ -0,0 +1,88 @@
+{% extends "./layouts/default.layout.njk" %}
+
+{% set pageName="Hmcts - Apply to adopt a child" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ pageHeading("Apply to adopt a child placed in your care") }}
+
+      <ol class="task-list">
+        <li>
+          <h2 class="task-list__section">
+            <span class="task-list__section-number">1. </span> Add application details
+          </h2>
+          {{ subTaskList(
+            subTasks=(tasks.applicantDetails),
+            sectionId='application_details'
+          ) }}
+        </li>
+        <li>
+          <h2 class="task-list__section">
+            <span class="task-list__section-number">2. </span> Add your details
+          </h2>
+          <div class="alert alert--default">
+            <p class="margin-top--xs">You'll need to upload your:</p>
+            <ul class="govuk-list govuk-list--bullet">
+              <li>marriage or civil partnership certificate [if couple; only show for 1 user]</li>
+              <li>passport's photo page</li>
+              <li>driving licence</li>
+            </ul>
+            <p>If relevant to you, you'll also need to upload: </p>
+            <ul class="govuk-list govuk-list--bullet">
+              <li>your UK visa</li>
+              <li>immigration stamps in your passport</li>
+              <li>evidence of any name changes, for example deed pool</li>
+            </ul>
+          </div>
```

Comment:
@KiranVarma02 needs discussed. I've created an alert macro but with the amount of content needed here, the macro is of no use. Have you seen macro's that get injected HTML?